### ISSUE TITLE
Display irods path for cut and paset in the browser

### ISSUE THREAD

**Issue by burnce** [DICE-UNC/irods-cloud-browser#174]
Hello,

We have been using the browser for some time now and are beginning to see use cases where users wish to refer to a data object they have discovered in the browser or access it from R or the icommands. Unfortunately, there does not appear to be a way to obtain the path in a usable form. The boxes at the top of the screen cannot be cut and pasted properly and are truncated. The path is present in the URL, but it is encoded to escape special characters such as slash or space.

Would it be possible to implement the ability to cut and paste of one or multiuple iRODS path(s) out of the browser?

Thanks,

Burnce

**Comment by michael-conway** [comment]
Yes I agree that's a needed feature.  We can push that forward in 
priority in this milestone we are working
